Gemfile

```diff
@@ -14,7 +14,7 @@ end
 
 # bundler is built into Ruby 2.6 and later
 if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.6')
-  gem 'bundler', '~> 1.11'
```

Comment:
Do we even need to set a floor here?

Reply:
i'm not sure it even matters at all?

Reply:
https://github.com/inspec/train/pull/98 is why the floor is there.

Reply:
yeah this clearly doesn't make any sense going forwards since in ruby >= 2.6 this line has to be avoided entirely anyway...